openstack/swift/templates/servers-daemonset.yaml

```diff
@@ -10,6 +10,8 @@ metadata:
 spec:
   updateStrategy:
     type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 2 # each object has 3 replicas, so we can afford to have 2 servers missing at a time without causing outage
```

Comment:
The only problematic situation, would be `HEAD` requests for non existent objects, which would result in a `503`. But I think we can accept this.